Description: Disable not packaged yet crc64
Last-Update: 2023-12-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -35,2 +35,2 @@
-[dev-dependencies.crc64]
-version = "2"
+#[dev-dependencies.crc64]
+#version = "2"
--- a/src/test.rs
+++ b/src/test.rs
@@ -101,7 +101,7 @@
     assert_eq!( 64, to_ansi(( 97, 134,   8)));
 }
 
-/// Calculates RGB→ANSI for all colours and calculates a checksum of them
+/*// Calculates RGB→ANSI for all colours and calculates a checksum of them
 /// comparing it to known value.  This is meant to see whether refactoring of
 /// the code does not change the behaviour.  If the computation is changed on
 /// purpose simply update the checksum in this test.
@@ -116,4 +116,4 @@
         }
     }
     assert_eq!(3373856917329536106, checksum);
-}
+}*/
